client/capstone-app/src/components/SearchResults.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { parseSearchData } from "../../utils";
+
+const SearchResults = ({ data, setCurrentLocation, setDestination, setCurrentCoordinates, setDestinationCoordinates }) => {
+  console.log(data)
+  const parsedData = parseSearchData(data)
+
+  if (setCurrentLocation) {
+    return (
+      <div>
```

Comment:
Use <></> instead of <div></div> if there is no styling in react. This is react convention.